pmdarima/arima/auto.py

```diff
@@ -96,6 +97,7 @@ def __init__(self, start_p=2, d=None, start_q=2, max_p=5,
         self.scoring = scoring
         self.scoring_args = scoring_args
         self.with_intercept = with_intercept
+        self.sarimax_kwargs = dict() if not sarimax_kwargs else sarimax_kwargs
```

Comment:
Same comment regarding `dict()` vs `{}` (just respond to one, just wanted to point both out)